### DISCUSSION TITLE
How to custom or override theme-common.json ?

### DISCUSSION THREAD

**Discussion by namnguyenthanhwork** [facebook/docusaurus#8973] [Q&A] [answered]
I have a question, i want to custom (or override) translate in my website in addition to using the default translation. How to do it ? 
Thanks everyone for your help.
<img width="1501" alt="image" src="https://github.com/facebook/docusaurus/assets/78526193/7c9aa686-d44c-4e57-824d-0bc2d93f0274">

**Comment by Josh-Cena** [comment, marked as answer]
Please see https://docusaurus.io/docs/i18n/tutorial#translate-your-react-code: you run `yarn write-translations` to generate the JSON file which you can edit yourself.

**Comment by namnguyenthanhwork** [comment]
Thank you @Josh-Cena
